AE-2334 review templates (#24)
OpenZeppelin/defender-templates: defender/refill-tokens/README.md

> # Refill tokens template

Reviewer: ```suggestion
# Refill Tokens Template
```

> Takes some address that is being monitored for some particular token balance on a scheduled time basis. If the balance is below a value defined as maximum, the Autotask will attempt to top-up the address with that token.

Reviewer: ```suggestion
Monitors an address for the balance of a specific token, checking on a pre-defined schedule. If the balance is below a value defined as maximum, the Autotask will attempt to top-up the address with that token.
```
